R/volcanoplot.R

```diff
@@ -230,7 +241,7 @@ srv_g_volcanoplot <- function(id,
 
         card$append_text(final_encodings, style = "verbatim")
         card$append_text("Plot", "header3")
-        card$append_plot(plot_r())
+        card$append_plot(plot_r(), dim = pws_p$dim())
         if (isTRUE(input$show_top_gene)) {
           card$append_text("Table", "header3")
           card$append_table(show_top_gene_diffexpr())
```

Comment:
reporter supports data.frame and rtables at the same time.

Reply:
ok that is great, good to know!